src/docs/references/03-email-templates/index.md

```diff
@@ -331,22 +331,28 @@ Context for transaction emails:
     "id": "uuid",
     "first-name": "string",
     "last-name": "string",
-    "display-name": "string"
+    "display-name": "string",
+    "private-data": "extended-data",
+    "public-data": "extended-data",
+    "protected-data": "extended-data"
   },
   "marketplace": {
     "name": "string",
-    "url" "string"
+    "url": "string"
   },
   "recipient-role": {"one-of": ["provider", "customer"]},
   "other-party": {
     "id": "uuid",
     "first-name": "string",
     "last-name": "string",
-    "display-name": "string"
+    "display-name": "string",
+    "private-data": "extended-data",
+    "public-data": "extended-data",
+    "protected-data": "extended-data"
   },
   "transaction": {
     "id": "uuid",
-    "line-items": [
+    "tx-line-items": [
```

Comment:
Has this changed or was the reference wrong?

Reply:
@kpuputti It has changed.

Reply:
Ok, good to know, just made sure. I assume all the deprecation, possible braking changes and other stuff is taken into account.

Reply:
Yep, so `line-items` is still available in the context but not documented anywhere.